### ISSUE TITLE
Autocomplete doesn't suggest names in import statement

### ISSUE THREAD

**Issue by serin-delaunay** [nwolverson/atom-ide-purescript#126]
I have the following Main.purs:

```
module Main where

import Prelude

main :: Int -> Int
main x = 4 * x
```

If I type `import Control.Monad.Eff.Console (` I expect autocomplete to suggest names to import from `Control.Monad.Eff.Console`, but nothing appears.
This also happens if I start with 

```
module Main where

import Prelude
import Control.Monad.Eff.Console

main :: Int -> Int
main x = 4 * x
```

and type `(` after the `Console` import.

**Comment by kritzcreek** [comment]
With psc-ide you shouldn't have to manually edit your explicit import statements anymore. Instead you should turn on the `Add import on autocomplete` setting, type out the function you are looking for, select it from the dropdown and your import should be properly inserted. Make sure you only have a single open import for `Prelude` and the plugin will always insert explicit imports for your other modules.

**Comment by nwolverson** [comment]
@kRITZCREEK certainly it's not the way I have of working, but I certainly think it's a valid issue, it would be quite reasonable to autocomplete in import lists).

**Comment by nwolverson** [comment]
Implemented ac31d7ffed2e35e1aa4d6775de54f6836b7f2ecb
